package.json

```diff
@@ -51,31 +51,30 @@
     "common/**"
   ],
   "dependencies": {
-    "globals": "^15.4.0",
-    "node-addon-api": "^8.0.0",
-    "node-gyp-build": "^4.8.1"
+    "globals": "^15.9.0",
+    "node-addon-api": "^8.1.0",
+    "node-gyp-build": "^4.8.2"
   },
   "peerDependencies": {
     "tree-sitter": "^0.21.1"
   },
   "peerDependenciesMeta": {
-    "tree_sitter": {
+    "tree-sitter": {
       "optional": true
     }
   },
   "devDependencies": {
-    "eslint": "^9.4.0",
+    "eslint": "^9.9.1",
     "eslint-config-google": "^0.14.0",
     "prebuildify": "^6.0.1",
     "tree-sitter-cli": "^0.23.0"
   },
   "scripts": {
     "install": "node-gyp-build",
-    "prebuildify": "prebuildify --napi --strip",
-    "generate": "for dir in php php_only; do cd $dir && tree-sitter generate --no-bindings && cd ..; done",
```

Comment:
Perhaps I'm doing something wrong/inefficient, but I use this `npm run generate` command to quickly generate both parsers. Is there a better one-liner command to generate the parsers when I'm trying to test my changes?

Reply:
oh, honestly that might be fine to leave in there. the rationale is that the scripts in here should *solely* pertain to the npm bindings/js files

Reply:
(feel free to add that one back, I get that it's annoying having to run that manually in multi-grammar repos)

Reply:
Actually, I suppose the makefile could be used for this, `make generate` or just `make` works just fine for me

Reply:
yeah that would probably be better actually - feel free to update it or I can in a bit

Reply:
Done :smile: